contracts/Manager.sol

```diff
@@ -63,7 +63,7 @@ contract Manager is Ownable {
     // ROLES
 
     // Manager is already Ownable, but in addition it also has an `operator`.
-    address operator;
+    address public operator;
```

Comment:
Are there arguments for this being `private`? OpenZeppelin had `_owner` as `private` in `Ownable.sol`, for example.

Reply:
Either way we should specify explicitly.